dotCMS/src/main/webapp/dotcms-webcomponents/p-kqyjznbj.system.entry.js

```diff
@@ -0,0 +1 @@
his.required,name:this.name},s("div",{"aria-describedby":c(this.hint),tabIndex:this.hint?0:null,class:"dot-tags__container"},s("dot-autocomplete",{class:d(this.status.dotValid),data:this.data,debounce:this.debounce,disabled:this.isDisabled(),onEnter:this.onEnterHandler.bind(this),onLostFocus:this.blurHandler.bind(this),onSelection:this.onSelectHandler.bind(this),placeholder:this.placeholder||null,threshold:this.threshold}),s("div",{class:"dot-tags__chips"},this.getValues().map((function(e){return s("dot-chip",{disabled:t.isDisabled(),label:e,onRemove:t.removeTag.bind(t)})}))))),l(this.hint),h(this.showErrorMessage(),this.getErrorMessage()))};t.prototype.addTag=function(t){var e=this.getValues();if(!e.includes(t)){e.push(t);this.value=e.join(",");this.updateStatus();this.emitChanges()}};t.prototype.blurHandler=function(){if(!this.status.dotTouched){this.status=p(this.status,{dotTouched:true});this.emitStatusChange()}};t.prototype.emitChanges=function(){this.emitStatusChange();this.emitValueChange()};t.prototype.emitStatusChange=function(){this.dotStatusChange.emit({name:this.name,status:this.status})};t.prototype.emitValueChange=function(){this.dotValueChange.emit({name:this.name,value:this.value})};t.prototype.getErrorMessage=function(){return this.isValid()?"":this.requiredMessage};t.prototype.getValues=function(){return f(this.value)?this.value.split(","):[]};t.prototype.isDisabled=function(){return this.disabled||null};t.prototype.isValid=function(){return!this.required||this.required&&!!this.value};t.prototype.onEnterHandler=function(t){var e=this;var i=t.detail,s=i===void 0?"":i;s.split(",").forEach((function(t){e.addTag(t.trim())}))};t.prototype.onSelectHandler=function(t){var e=t.detail,i=e===void 0?"":e;var s=i.replace(","," ").replace(/\s+/g," ");this.addTag(s)};t.prototype.removeTag=function(t){var e=this.getValues().filter((function(e){return e!==t.detail}));this.value=e.join(",");this.updateStatus();this.emitChanges()};t.prototype.showErrorMessage=function(){return this.getErrorMessage()&&!this.status.dotPristine};t.prototype.updateStatus=function(){this.status=p(this.status,{dotTouched:true,dotPristine:false,dotValid:this.isValid()})};t.prototype.validateProps=function(){this.valueWatch()};Object.defineProperty(t.prototype,"el",{get:function(){return a(this)},enumerable:true,configurable:true});Object.defineProperty(t,"watchers",{get:function(){return{value:["valueWatch"]}},enumerable:true,configurable:true});Object.defineProperty(t,"style",{get:function(){return"dot-tags .dot-tags__container{display:-ms-flexbox;display:flex;-ms-flex-align:start;align-items:flex-start;border:1px solid #d3d3d3}dot-tags .dot-tags__container dot-autocomplete{margin:.5rem 1rem .5rem .5rem}dot-tags .dot-tags__container .dot-tags__chips{margin:.5rem 1rem 0 0}dot-tags .dot-tags__container dot-chip{border:1px solid #ccc;display:inline-block;margin:0 .5rem .5rem 0;padding:.2rem}dot-tags button{border:0}"},enumerable:true,configurable:true});return t}())}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5062311)